docs/how-to/manage-passwords.md

```diff
@@ -0,0 +1,27 @@
+(manage-passwords)=
+# How to manage passwords
+
+In the [Access etcd](#access-etcd) section of the tutorial, we didn't need to include a password in the HTTP request. However, in order to read or write data in etcd, we need to authenticate ourselves.
+
+Typically, this can be done using a username and TLS certificate. For the sake of this guide, we will also use charmed etcd's internal admin user `root`. This user is only for  internal use, and it is created automatically by charmed etcd.
+
+We will go through setting a user-defined password for the admin user and configuring it to charmed etcd. 
+
+First, create a secret in `Juju` containing your password. You will get the `secret` ID as a response.
+
+```{terminal}
+:input: juju add-secret mysecret root=changeme
+
+secret:ctbirhuutr9sr8mgrmpg
+```
+Make note of the string following `secret:`.
```

Comment:
Suggestion: Here and later add empty lines before and after code blocks. That is a Markdown best practice to do so.